CMakeLists.txt

```diff
@@ -142,6 +142,10 @@ message(" ")
 
 include(SSGCommon)
 
+# Build shared CPE first
```

Comment:
Yeah, I realized that when I started building hehehe, I changed the comment to "Configuring"